src/js/controller/controller.js

```diff
@@ -309,6 +309,8 @@ Object.assign(Controller.prototype, {
             });
 
             preload();
+
+            _checkPauseOnViewable(model, viewable);
```

Comment:
Is this now the only place this is called? Why are you passing in the model? At this point this should be it's own callback for "change: viewable" only added when `autoPause.viewability` is `true`.